### ISSUE TITLE
New v5 apiKey.

### ISSUE THREAD

**Issue by fotrosi** [burakoner/OKEx.Net#21]
Hi,
I have this error:
"30016: Please change to port v1 when you are using v1 apiKey. If you wish to use port v3, please register a v3 apiKey"
My apiKey version is v5 and I can't create v3 apiKey. Please help me.
Thx.

**Comment by EgorovOleg** [comment]
> Hi,
> I have this error:
> "30016: Please change to port v1 when you are using v1 apiKey. If you wish to use port v3, please register a v3 apiKey"
> My apiKey version is v5 and I can't create v3 apiKey. Please help me.
> Thx.

Hi. you solved this problem?

**Comment by fotrosi** [comment]
> > Hi,
> > I have this error:
> > "30016: Please change to port v1 when you are using v1 apiKey. If you wish to use port v3, please register a v3 apiKey"
> > My apiKey version is v5 and I can't create v3 apiKey. Please help me.
> > Thx.
> 
> Hi. you solved this problem?

Hi, No, this problem still exists.

**Comment by EgorovOleg** [comment]
> > > Hi,
> > > I have this error:
> > > "30016: Please change to port v1 when you are using v1 apiKey. If you wish to use port v3, please register a v3 apiKey"
> > > My apiKey version is v5 and I can't create v3 apiKey. Please help me.
> > > Thx.
> > 
> > 
> > Hi. you solved this problem?
> 
> Hi, No, this problem still exists.

Sometimes I get the v3 key generation menu. And the menu becomes adding api v5.
![image](https://user-images.githubusercontent.com/47451430/116224578-79a95300-a759-11eb-8ab2-ebf9dbcbaa93.png)

**Comment by MsLolita** [comment]
Somebody solve this error?

**Comment by EgorovOleg** [comment]
> Somebody solve this error?
What error?

**Comment by ikaman** [comment]
for me the same issue.

Anyone had a fix for this? does this mean the whole lib has to be updated to api v5? (as you cannot make api v3 keys any more)

**Comment by MsLolita** [comment]
> for me the same issue.
> 
> Anyone had a fix for this? does this mean the whole lib has to be updated to api v5? (as you cannot make api v3 keys any more)

Yeah, there isn't anyway to get v3 apikey, so use only v5 documentation with v5 keys.

**Comment by ikaman** [comment]
> > for me the same issue.
> > Anyone had a fix for this? does this mean the whole lib has to be updated to api v5? (as you cannot make api v3 keys any more)
> 
> Yeah, there isn't anyway to get v3 apikey, so use only v5 documentation with v5 keys.

Can you explain a little bit more pls? How can I use only v5 documentation? It seems whatever endpoint I use I keep getting the same error from the start of the thread

**Comment by claudius6** [comment]
I solved it!

You guys need to use the actual new docs which is v5: https://www.okex.com/docs-v5/en/#overview

If you are using sdk in python or others. change the Links there!

They changed the links completely. e.g. the old format was like: /api/account/v3/wallet but the new format is different e.g. /api/v5/account/balance

**Comment by ikaman** [comment]
how do you use the new docs? I didnt see an option in the SDK to do that

**Comment by ikaman** [comment]
> I solved it!
> 
> You guys need to use the actual new docs which is v5: https://www.okex.com/docs-v5/en/#overview
> 
> If you are using sdk in python or others. change the Links there!
> 
> They changed the links completely. e.g. the old format was like: /api/account/v3/wallet but the new format is different e.g. /api/v5/account/balance
> 
> you can change that at okex.consts.py

how do you use the new docs? I didnt see an option in the SDK to do that

**Comment by claudius6** [comment]
> > I solved it!
> > You guys need to use the actual new docs which is v5: https://www.okex.com/docs-v5/en/#overview
> > If you are using sdk in python or others. change the Links there!
> > They changed the links completely. e.g. the old format was like: /api/account/v3/wallet but the new format is different e.g. /api/v5/account/balance
> > you can change that at okex.consts.py
> 
> how do you use the new docs? I didnt see an option in the SDK to do that

You need to rewrite some functions. take a look inside the funtion they use and change it that it fits with the the new docs

**Comment by claudius6** [comment]
its enough to change the parameters names and the links. (links at consts.py) like explained above
